src/main/java/io/cdap/plugin/google/drive/source/SchemaBuilder.java

```diff
@@ -27,7 +27,7 @@
  * Util class for building pipeline schema.
  */
 public class SchemaBuilder {
-  public static final String SCHEMA_ROOT_RECORD_NAME = "FileFromFolder";
+  public static final String SCHEMA_ROOT_RECORD_NAME = "etlSchemaBody";
```

Comment:
why was this change needed?

Reply:
The schema was not being updated in _Google Drive Source_ and initially we thought that this was the case, but then we found out that the issue was related with widget, not with `SCHEMA_ROOT_RECORD_NAME`. Reverted to its initial name in the updated PR.